packages/pn-commons/src/components/Data/ItemsCard/ItemsCardContent.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { Fragment } from 'react';
+
+import { Box, Typography } from '@mui/material';
+
+import { CardElement } from '../../../models';
+
+export interface IItemsCardContentProps {
+  body: CardElement;
+  children?: React.ReactNode;
+}
+
+const ItemsCardContent: React.FC<IItemsCardContentProps> = ({ body, children }) => (
```

Comment:
Due to the change requests on the parent component, this component  will change in this way:
- `{(!body.hideIfEmpty || (body.hideIfEmpty && children))` will be useless and managed from the outside in the standard way (i.e. {value_exists && component_that_print_value})
- an additional property type (value | label) will be added and the behaviour of the component changes based on its value
- notWrappedInTypography becomes wrappedInTypography with default value set to true
- div will be replaced by Box